scheduler/src/cook/mesos/task.clj

```diff
@@ -80,9 +80,9 @@
 (defn merge-container-defaults
   "Takes a job container specification and applies any defaults from the config.
    Currently only supports volumes."
-  [container]
+  [container compute-cluster]
```

Comment:
Nitpick: can we make `compute-cluster` the first argument? Keeping the more 'static' value as the first arguments helps with currying.

Reply:
Good call, done